### DISCUSSION TITLE
openradioss under windows: how to generate h3d + how to generate vtk files

### DISCUSSION THREAD

**Discussion by paul18frr** [OpenRadioss/OpenRadioss#807] [Users]
Hi,

For a test, i ran a radioss file i used years ago) but under the new Windows release; it worked fine for the starter and the engine steps; nonetheless:

- the h3d file has been generated : how to generate it? (maybe i'm mistaken but it was automatically generated under radioss)
- I've not figured out so far how to create paraview files using "anim_to_vtk_win64" exe? its asking for a file as input, but which one? is it based on the h3d file?

Thanks for your support

**Comment by sebastienVilleneuve** [comment]
Hi,

- h3d files are generated using /H3D keywords in 1.rad file
- input for anim_to_vtk is an animation file ( rootnameAxxx file)

Regards,
Sébastien

**Reply by paul18frr** [reply]
Thanks Sebastien fr the answer.

Indeed the /H3D keyword was missing in he _01.rad file, so  the behaviour is obvious.

Finally I figured out how to use the vtk translator; I need to write a script to automate it

Thanks for the support

**Reply by ceanwang** [reply]
> * h3 files are generated using /H3D keywords in 1.rad file

Under Windows 10, I am trying the Spring-back demo case from here: https://openradioss.atlassian.net/wiki/spaces/OPENRADIOSS/pages/9502721/Spring-back

At the end of DBEND_44_0001.rad, I added /H3D

When run:
`engine_win64 -i dbend_44_0001.rad` 
I got an msg saying: 
`MESSAGE ID :        275
 ** ERROR: NOT EXISTING /H3D OPTION: /H3D/                    /                    /                    /                    /
     ERROR TERMINATION`
Any idea?
Regards
Cean

**Reply by sebastienVilleneuve** [reply]
Hi,

I let you check the online help , available /H3D keywords are : 

![image](https://user-images.githubusercontent.com/100404030/203502977-d34e590c-807c-4f80-ba77-e86cb7863556.png)

at least you need the /H3D/DT keyword to define the output time step 

Sébastien

**Comment by paul18frr** [comment]
I've an additional question: in my curent case, element erosion is "activated", and I'm wondering if eroded elements have been removed in the vtk file? Regarding the screenshot, I feel it's not the case, am i right?  otherwise how to explain such behaviour?

i'm not a paraview user (not yet :-) ), but maybe it's easy to create a filter to remove them at each time: does somebody has a feedback on it?

In the meantime, the test case will be rerun adding the h3D keyword in order to compare final views with h3d viewer.

Paul

![export](https://user-images.githubusercontent.com/54111594/203008464-82dce207-6a57-4ca6-96f6-66e547e9de10.png)

**Reply by PaulAltair** [reply]
the eroded elements have a flag 'EROSION_STATUS' this is converted in the vtk. Hyperview hides the eroded elements automatically, in paraview you need to add a filter on that flag: e.g. 
![image](https://user-images.githubusercontent.com/98418895/203027691-3ffb9e14-ded3-4fbc-8a8e-6494262fdce6.png)

**Reply by paul18frr** [reply]
Thanks
it works now as expected
Paul
